File: chao_node.py
#!/usr/bin/env python3
#coding=utf-8

import rospy
from std_msgs.msg import String
from qq_msgs.msg import Carry

if __name__=="__main__":
    rospy.init_node("chao_node")
    rospy.logwarn("我的枪去而复返，你的生命有去无回！")

    pub = rospy.Publisher("kuai_shang_che_kai_hei_qun",Carry,queue_size=10)

    rate = rospy.Rate(10)

    while not rospy.is_shutdown():
        rospy.loginfo("我要开始刷屏了")
        msg = Carry()
        msg.grade = "王者"
        msg.star = 50
        
        msg.data = "国服马超，带飞"
        pub.publish(msg)
        rate.sleep()

File: chao_node.cpp
#include<ros/ros.h>
#include<std_msgs/String.h>
#include<qq_msgs/Carry.h>

int main(int argc,char *argv[])
{
    ros::init(argc,argv,"chao_node");
    printf("我的枪去而复返，你的生命有去无回！\n");

    ros::NodeHandle nh;
    ros::Publisher pub = nh.advertise<qq_msgs::Carry>("kuai_shang_che_kai_hei_qun",10);

    ros::Rate loop_rate(10);

    while(ros::ok())
    {
        printf("我要开始刷屏了!\n");
        qq_msgs::Carry msg;
        msg.grade = "王者";
        msg.star = 50;
        
        msg.data="国服马超，带飞";
        pub.publish(msg);
        loop_rate.sleep();
    }
    return 0;
}
